readme.md

```diff
@@ -15,11 +15,17 @@ $ npm install --save dot-prop
 ```js
 var dotProp = require('dot-prop');
 
+// getter
 dotProp({foo: {bar: 'unicorn'}}, 'foo.bar');
 //=> 'unicorn'
 
-dotProp({foo: {bar: 'a'}}, 'foo.notDefined.deep');
```

Comment:
@stevemao theories... theories... anyway this is clearly a regression - a lot of people may use this module because of this feature. Throwing an exception should be discussed in separate PR.